test/lib/sign.js

```diff
@@ -4,6 +4,8 @@ const path = require('path')
 const mockery = require('mockery')
 
 const ChildProcessMock = require('../fixtures/child_process')
+const { isValidPublisher, signAppx } = require('../../lib/sign')
```

Comment:
Looks like node 4.x CI is upset about this. Would you prefer the I make this work with 4.x or update to require node 6.x? (4.x enters maintenance phase in a few days then EOL next year)